x-pack/plugins/observability_solution/infra/server/lib/alerting/log_threshold/register_log_threshold_rule_type.ts

```diff
@@ -173,7 +181,10 @@ export async function registerLogThresholdRuleType(
       extractReferences,
       injectReferences,
     },
-    alerts: LogsRulesTypeAlertDefinition,
+    alerts: {
```

Comment:
should be able to use `alerts: LogsRulesTypeAlertDefinition` after setting `shouldWrite: true` there.

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/178680/commits/caa163f3d5455b6a09160abc491d9a870fcd208f